changelog.md

```diff
@@ -12,6 +17,11 @@
     + inflection@1.6.0
     + lodash@3.5.0
     + validator@3.34
+    + generic-pool@2.2.0
+- [INTERNALS] Updated devDependencies.
```

Comment:
Well, these were missing in the changelog. I know it's not nice to change these after releasing but I guess it's always better for people looking things up later on.